scripts/convert_original_stable_diffusion_to_diffusers.py

```diff
@@ -100,6 +100,26 @@
     )
     parser.add_argument("--dump_path", default=None, type=str, required=True, help="Path to the output model.")
     parser.add_argument("--device", type=str, help="Device to use (e.g. cpu, cuda:0, cuda:1, etc.)")
+    parser.add_argument(
+        "--stable_unclip",
+        type=str,
+        default=None,
+        required=False,
+        help="Set if this is a stable unCLIP model. One of 'txt2img' or 'img2img'.",
+    )
+    parser.add_argument(
+        "--stable_unclip_prior",
+        type=str,
+        default="karlo",
```

Comment:
```suggestion
        default=None,
```

Shouldn't default be `None`?

Reply:
done here ! https://github.com/huggingface/diffusers/pull/2297/commits/e3458b0b1a8b63e23cdf49e07efe483a7dd6cb91